pages/common/gitlint.md

```diff
@@ -0,0 +1,28 @@
+# gitlint
+
+> Git commit message linter checks your commit messages for style.
+> More information: <https://jorisroovers.com/gitlint/>.
+
+- Show usage and optional arguments:
+
+`gitlint --help`
```

Comment:
Same applies here.

Reply:
Similarly here also i can remove or replace these, will it be fine?

Reply:
Yes.